button.js

```diff
@@ -33,9 +39,15 @@ function injectToggle(revPromise, path) {
     enabled = !enabled;
     if (enabled) {
       button.appendChild(spinner);
-      await applyOverlay(revPromise, path);
-      button.removeChild(spinner);
-      button.style.backgroundColor = 'lightgrey';
+      try {
+        await applyOverlay(revPromise, path);
+        button.removeChild(spinner);
+        button.style.backgroundColor = 'lightgrey';
+      } catch (ex) {
+        button.style.backgroundColor = 'red';
+        button.removeChild(spinner);
+        disableButton(button, 'Error retrieving coverage information for this file');
+      }
```

Comment:
Since `button.removeChild(spinner);` is shared between the try and the catch, maybe you can move it to the finally clause:
```JavaScript
try {
  ...
catch (ex) {
  ...
} finally {
  button.removeChild(spinner);
}
```